src/include/sof/math/log.h

```diff
@@ -0,0 +1,21 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2021 Intel Corporation. All rights reserved.
+ *
+ * Author: Shriram Shastry <malladi.sastry@linux.intel.com>
+ *
+ */
+
+#ifndef __SOF_MATH_BASE_LOGARITHM_H__
+#define __SOF_MATH_BASE_LOGARITHM_H__
+
+#include <stdint.h>
+
+#define ONE_OVER_LOG2_10	0x268826A2000ULL	/* 1/log2(10) */
+#define ONE_OVER_LOG2_E		0x2C5C86D7C93ULL	/* 1/log2(exp(1)) */
```

Comment:
And maybe this is Q1.42? Or Q0.42 unsigned. I don't recommend to use hex constant for numbers those do not consist of bitfields (like I/O registers bits in device drivers). It's just extra effort to try to figure out the number. Decimal is more convenient and should work as well as unsigned long long.

Reply:
Sure,